wakusim.env

```diff
@@ -1,14 +1,15 @@
 # Env variables for metal-01.he-eu-hel1.wakusim.misc host.
-NWAKU_IMAGE=wakuorg/nwaku:latest
-GOWAKU_IMAGE=wakuorg/go-waku:v0.8.0
```

Comment:
can you rebase? afaik we removed this

Reply:
this is also still present in master, I forgot to add this change in the PR that removes the gowaku support

Reply:
The parameters are removed in this PR https://github.com/waku-org/waku-simulator/pull/47